social_core/backends/facebook.py

```diff
@@ -59,6 +58,7 @@ def get_user_details(self, response):
         )
         return {'username': response.get('username', response.get('name')),
                 'email': response.get('email', ''),
+                'profile_picture': response.get('profile_picture', ''),
```

Comment:
I don't think this should be added.  `get_user_details` is just to translate each authentication services response into a unified format.  The closest I could get to docs, though, was this:  https://github.com/omab/python-social-auth/issues/7#issuecomment-18087704